src/components/Reporting/ReportingQueryModals.tsx

```diff
@@ -0,0 +1,172 @@
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleAdvisorIdChange = (e: any): void => {
+    setAdvisorId(e.target.value);
+    localStorage.setItem("advisorId", e.target.value);
+  };
+
+  const removeLocalStorageItems = () => {
+    localStorage.removeItem("advisorId");
+    localStorage.removeItem("startDate");
+    localStorage.removeItem("endDate");
+  };
+
+  const submitHandler: React.FormEventHandler = async (event) => {
+    event.preventDefault();
+    setIsLoading(true);
+    const dataToSend = {
+      accountId: localStorage.getItem("advisorId"),
+      startDate: localStorage.getItem("startDate"),
+      endDate: localStorage.getItem("endDate"),
+      email: localStorage.getItem("email"),
+    };
+    axios
+      .post(URL, dataToSend, {
+        headers: {
+          "x-auth-token": `${localStorage.getItem("token")}`,
+        },
+      })
+      .then(function (response) {
+        setIsLoading(false);
+        setSubmitError(false);
+        removeLocalStorageItems();
+        console.log(response);
+      })
+      .catch(function (error) {
+        setIsLoading(false);
+        setSubmitError(true);
+        removeLocalStorageItems();
+        console.log(error);
+      });
+    setSubmitButtonClicked(true);
+  };
+
+  return (
+    <>
+      <Modal>
+        <Box className={classes.paper}>
+          <Box display="flex" justifyContent="flex-end">
+            <SlClose
+              onClick={() => {
+                closeHandler();
+              }}
+              style={{ cursor: "pointer" }}
+            />
+          </Box>
+          <Typography variant="h4" id="modal-title">
+            {selectedQuery.name} Report
+          </Typography>
+
+          {!submitButtonClicked && (
+            <form action="submit" onSubmit={submitHandler}>
+              <TextField
+                required
+                type="string"
+                label="Enter Account ID"
+                value={advisorId}
+                className={classes.textField}
+                onChange={handleAdvisorIdChange}
+                style={{ marginTop: "16px", marginLeft: "8px" }}
+                InputLabelProps={{
+                  className: classes.labelColor,
+                }}
+              />
+
+              <DatePickers />
+              <Button
+                variant="contained"
+                size="small"
+                style={{ marginTop: "20px", marginLeft: "5px" }}
+                type="submit"
+              >
+                Send me email
+              </Button>
+            </form>
+          )}
+
+          {isLoading && <Loader />}
+
+          {!isLoading && submitButtonClicked && !submitError && (
+            <Box
+              style={{
+                marginTop: "16px",
+                color: "#008000",
+                fontWeight: "bold",
+              }}
+            >
+              Thank you! Report will be sent to your email soon!
```

Comment:
lets slightly rephrase to 
"Thank you! The requested report will be sent to your email soon!"